quartz/components/renderPage.tsx

```diff
@@ -213,6 +216,17 @@ export function renderPage(
   )
 
   const lang = componentData.fileData.frontmatter?.lang ?? cfg.locale?.split("-")[0] ?? "en"
+
+  let content = <Content {...componentData} />
+  if (cfg.enablePassProtected && componentData.fileData.frontmatter?.password) {
+    componentData.iteration = 2e6
```

Comment:
type for QuartzComponentProps also need to be updated with this. (for both iteration and encryptedContent

Reply:
I moved `iteration` to global config.
And updated types of QuartzComponentProps for `encryptedContent`.